docs/setting-up/client/docker.md

```diff
@@ -1,96 +1,57 @@
-```
-
-!!! note
-
-    This container does not run, but exists only to make sure you retain
-all PMM data when upgrading to a newer image.
-
-* The `-v` option initializes a data volume for the container.
-
-* The `--name` option assigns a name for the container
-to reference the container within a Docker network.
+You use it to create a Docker container for your PMM Client.
 
-* `percona/pmm-client:2` is the name and version tag of the image
-to derive the container from.
+When launched, the Docker container gives access to the whole functionality of PMM Client.
 
-* `/bin/true` is the command that the container runs.
+## Running PMM Client as a Docker container
 
-### Run the PMM Client Docker Container
+1. Pull the image
 
-```sh
-docker run --rm \
-    -e PMM_AGENT_SERVER_ADDRESS=PMMServer:443 \
-    -e PMM_AGENT_SERVER_USERNAME=admin \
-    -e PMM_AGENT_SERVER_PASSWORD=admin \
-    -e PMM_AGENT_SERVER_INSECURE_TLS=1 \
-    -e PMM_AGENT_SETUP=1 \
-    -e PMM_AGENT_CONFIG_FILE=pmm-agent.yml \
-    --volumes-from pmm-client-data \
-    perconalab/pmm-client:2
-```
+        docker pull percona/pmm-client:2
 
-### ENVIRONMENT VARIABLES
+2. Create a persistent data store
 
-`PMM_AGENT_SERVER_ADDRESS`
-: The PMM Server hostname and port number.
+        docker create -v /srv --name pmm-client-data percona/pmm-client:2 /bin/true
 
-`PMM_AGENT_SERVER_USERNAME`
-: The PMM Server user name.
+    !!! alert alert-info "Note"
+        This container does not run, but exists only to make sure you retain all PMM data when upgrading to a newer image.
 
-`PMM_AGENT_SERVER_PASSWORD`
-: The PMM Server user’s password.
+3. Run the container
 
-`PMM_AGENT_SERVER_INSECURE_TLS`
-: If true (1), use insecure TLS. Otherwise, do not.
+        docker run --rm \
+            -e PMM_AGENT_SERVER_ADDRESS=PMMServer:443 \
+            -e PMM_AGENT_SERVER_USERNAME=admin \
+            -e PMM_AGENT_SERVER_PASSWORD=admin \
+            -e PMM_AGENT_SERVER_INSECURE_TLS=1 \
+            -e PMM_AGENT_SETUP=1 \
+            -e PMM_AGENT_CONFIG_FILE=pmm-agent.yml \
+            --volumes-from pmm-client-data perconalab/pmm-client:2
 
-`PMM_AGENT_SETUP`
-: If true (1), run `pmm-agent setup`. Default: false (0).
+!!! alert alert-success "Tip"
+    To get help: `docker run --rm perconalab/pmm-client:2 --help`
 
-`PMM_AGENT_CONFIG_FILE`
-: The PMM Agent configuration file.
+## Environment variables
 
-To get help:
+| Variable                        | Description   |
+| ------------------------------- | ------------- |
+| `PMM_AGENT_SERVER_ADDRESS`      | The PMM Server hostname and port number.
+| `PMM_AGENT_SERVER_USERNAME`     | The PMM Server user name.
+| `PMM_AGENT_SERVER_PASSWORD`     | The PMM Server user’s password.
+| `PMM_AGENT_SERVER_INSECURE_TLS` | If true (1), use insecure TLS. Otherwise, do not.
+| `PMM_AGENT_SETUP`               | If true (1), run `pmm-agent setup`. Default: false (0).
+| `PMM_AGENT_CONFIG_FILE`         | The PMM Agent configuration file.
```

Comment:
If you  run 
docker run --rm perconalab/pmm-client:2 --help
You can see the full list of parameters/variables available.  
Maybe we should include all of them?

Reply:
Good idea, I'll try.

Reply:
The command lists PMM Agent env vars. These are already described in the new pmm-agent man page. So I shall consolidate the env vars there and mention here that PMM agent envvars can be used.